### DISCUSSION TITLE
ROI Prior

### DISCUSSION THREAD

**Discussion by shubhambooking2** [google/meridian#154] [General]
In analyzing the return on investment (ROI) for media channels, I find that ROI estimates are heavily influenced by specified priors. Despite expecting the posterior distribution to be primarily influenced by the likelihood of data given a specific media channel, it appears that the priors still significantly impact the results. Even attempts to introduce non-informative priors, such as by increasing variance, result in some ROI estimates being unreasonably high (e.g., 50). How can I establish ROI priors in a manner that ensures the interpretation of signal from data rather than being disproportionately influenced by the priors?

**Comment by jyklly** [comment]
It seems like there may not be enough information in the data. Please see [When the posterior is the same as the prior](https://developers.google.com/meridian/docs/advanced-modeling/additional-considerations#posterior-same-as-prior) in the Additional Considerations section of the documentation as it addresses this issue. 

Please also see [Channels with low spend](https://developers.google.com/meridian/docs/advanced-modeling/additional-considerations#channels-with-low-spend), [When ROI results are widely different depending on the prior used
](https://developers.google.com/meridian/docs/advanced-modeling/additional-considerations#roi-results-widely-different) and [Amount of data needed](https://developers.google.com/meridian/docs/advanced-modeling/additional-considerations#general-amount-data-needed) for additional insights.
